tests/nested/manual/broken-model/task.yaml

```diff
@@ -47,9 +47,10 @@ execute: |
 
   # TODO:FDEM: also test degraded.json when we report what key was used.
 
-  if os.query is-ubuntu-ge 22.04; then
-    snap_bootstrap_service=snap-initramfs-mounts.service
-  else
-    snap_bootstrap_service=the-tool.service
-  fi
-  remote.exec "sudo journalctl -b0 -u ${snap_bootstrap_service}" | MATCH "successfully activated encrypted device .* using a fallback activation method"
+  #FIXME: use activation state instead
+  #if os.query is-ubuntu-ge 22.04; then
+  #  snap_bootstrap_service=snap-initramfs-mounts.service
+  #else
+  #  snap_bootstrap_service=the-tool.service
+  #fi
+  #remote.exec "sudo journalctl -b0 -u ${snap_bootstrap_service}" | MATCH "successfully activated encrypted device .* using a fallback activation method"
```

Comment:
I see the logs are intact under secboot_sb.go, should this check still be commented?